templates/user/edit-profile.html

```diff
@@ -148,7 +148,11 @@
                                 <td colspan="2"><span class="inline-header">
                                     {% if profile.is_totp_enabled %}
                                         {{ _('Two Factor Authentication is enabled.') }}
-                                        <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					                    {% if request.user.is_superuser %}
+					                        <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					                    {% else %}
+					                        <a href="javascript:alert('{{ _('The administrators for this site require all the staff to have two-factor authentication enabled, so it may not be disabled at this time.')|escapejs }}')" title="{{ _('The administrators for this site require all the staff to have two-factor authentication enabled, so it may not be disabled at this time.') }}" class="button inline-button">Disable</a>	
```

Comment:
way way way too long of a line, wrap it

Reply:
Still way too long and why are you mixing tabs and spaces?